metrics/web/lib/main.dart

```diff
@@ -112,3 +118,15 @@ class _MetricsAppState extends State<MetricsApp> {
     );
   }
 }
+
+/// A [ScrollBehavior] that disable the stretching effect.
+class _ScrollBehaviour extends ScrollBehavior {
```

Comment:
```suggestion
class MetricsScrollBehavior extends ScrollBehavior {
```
I think it also would be beneficial to move it to the `common/presentation/widgets` to make the `main.dart` more simple and to match Flutter's `ScrollBehavior` location (`src/widgets/scroll_configuration.dart`)